### PR TITLE
Fix broken Params documentation

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

**Comment by Geod24** [comment]
https://github.com/bpfkorea/agora/runs/662505048

File: source/geod24/LocalRest.d [hunk 1]
```diff
@@ -311,6 +311,8 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
           Impl = Type of the implementation to instantiate
           args = Arguments to the object's constructor
           timeout = (optional) timeout to use with requests
+          file = Path to the file that called this function (for diagnostic)
+          line = Line numbe tied to the `file` paramer
```
Comment (AndrejMitrovic) [inline source/geod24/LocalRest.d]:
```suggestion
          line = Line number tied to the `file` paramer
```

Comment (AndrejMitrovic) [reply source/geod24/LocalRest.d]: also paramer => parameter?

Comment (Geod24) [reply source/geod24/LocalRest.d]: *hides in shame*

**Comment by Geod24** [comment]
Fixed

**Review by AndrejMitrovic — APPROVED** [review]

**Pull request merged**